scrapers/freshmaker.py

```diff
@@ -18,7 +18,7 @@
 class FreshmakerScraper(BaseScraper):
     """Scrapes the Freshmaker API."""
 
-    freshmaker_url = "https://freshmaker.engineering.redhat.com/api/1/events/?per_page=50"
+    freshmaker_url = "https://freshmaker.engineering.redhat.com/api/2/events/?per_page=50"
```

Comment:
Optional: Could you use single quotes to match the rest of the codebase? I know you didn't introduce this, but it'd be nice to fix this since you're already modifying this.